tests/bluechi_test/container.py

```diff
@@ -60,7 +60,7 @@ def gather_journal_logs(self, data_dir: str) -> None:
 
     def cleanup(self):
         if self.container.status == 'running':
-            self.container.stop()
+            self.container.kill()
```

Comment:
I don't think `kill` is a good approach here as it doesn't allow for a graceful shutdown, thus creating zombie containers. These can not be removed and cleaned up properly - leading the subsequent tests to fail (if they use the same name for a container).

Reply:
From the tests, I ran locally,
tests timeouts and failures were reduced significantly
need to check

Reply:
OK just add the time to wait before kill in podman stop api

Reply:
Looks good!